tests/flow/test_procedures.py

```diff
@@ -76,194 +66,194 @@ def queryAndValidate(self, query, expected_results):
+
+
+        # CALL + WHERE + RETURN + SKIP + LIMIT.
+        query = """CALL db.idx.fulltext.queryNodes('fruit', 'Orange*') YIELD node
+                    WHERE node.value > 2
+                    RETURN node
+                    SKIP 1
+                    LIMIT 1"""
+        # not deterministic!
+        # self.queryAndValidate(query, expected_results)
+
+
+        # CALL + RETURN + ORDER.
+        query = """CALL db.idx.fulltext.queryNodes('fruit', 'Orange*') YIELD node
+                    RETURN node
+                    ORDER BY node.value
+                    """
+        expected_results = [node1, node2, node3, node4]
+        self.queryAndValidate(query, expected_results)
+
+
+        # CALL + RETURN + ORDER + SKIP.
+        query = """CALL db.idx.fulltext.queryNodes('fruit', 'Orange*') YIELD node
+                    RETURN node
+                    ORDER BY node.value
+                    SKIP 1
+                    """
+        expected_results = [node2, node3, node4]
+        self.queryAndValidate(query, expected_results)
+
+
+        # CALL + RETURN + ORDER + LIMIT.
+        query = """CALL db.idx.fulltext.queryNodes('fruit', 'Orange*') YIELD node
+                    RETURN node
+                    ORDER BY node.value
+                    LIMIT 2
+                    """
+        expected_results = [node1, node2]
+        self.queryAndValidate(query, expected_results)
+
+
+        # CALL + RETURN + ORDER + SKIP + LIMIT.
+        query = """CALL db.idx.fulltext.queryNodes('fruit', 'Orange*') YIELD node
+                    RETURN node
+                    ORDER BY node.value
+                    SKIP 1
+                    LIMIT 1
+                    """
+        expected_results = [node2]
+        self.queryAndValidate(query, expected_results)
+
+
+        # CALL + WHERE + RETURN + ORDER.
+        query = """CALL db.idx.fulltext.queryNodes('fruit', 'Orange*') YIELD node
+                    WHERE node.value > 2
+                    RETURN node
+                    ORDER BY node.value"""
+        expected_results = [node3, node4]
+        self.queryAndValidate(query, expected_results)
+
+
+        # CALL + WHERE + RETURN + ORDER + SKIP.
+        query = """CALL db.idx.fulltext.queryNodes('fruit', 'Orange*') YIELD node
+                    WHERE node.value > 2
+                    RETURN node
+                    ORDER BY node.value
+                    SKIP 1"""
+        expected_results = [node4]
+        self.queryAndValidate(query, expected_results)
+
+
+        # CALL + WHERE + RETURN + ORDER + LIMIT.
+        query = """CALL db.idx.fulltext.queryNodes('fruit', 'Orange*') YIELD node
+                    WHERE node.value > 2
+                    RETURN node
+                    ORDER BY node.value
+                    LIMIT 1"""
+        expected_results = [node3]
+        self.queryAndValidate(query, expected_results)
+
+
+        # CALL + WHERE + RETURN + ORDER + SKIP + LIMIT.
```

Comment:
Following the merge of #576, all clause combinations except any usage of WHERE should pass. This usage of WHERE is not supported by libcypher-parser; track progress at #577